app/workers/facilities/pssg_download.rb

```diff
@@ -31,28 +32,37 @@ def create_and_save_drive_time_data(drive_time_data)
       name = attributes&.dig('Name')
       drive_time_band = facility.drivetime_bands.find_or_initialize_by(vha_facility_id: id, name: name)
       drive_time_band.unit = 'minutes'
-      drive_time_band.min = attributes&.dig('FromBreak')
-      drive_time_band.max = attributes&.dig('ToBreak')
+      drive_time_band.min = round_band(attributes&.dig('FromBreak'))
+      drive_time_band.max = round_band(attributes&.dig('ToBreak'))
       drive_time_band.name = name
-      drive_time_band.polygon = extract_polygon(drive_time_data)
+
+      drive_time_band.polygon = extract_polygon(rings)
       drive_time_band.save
       facility.save
     end
 
-    def extract_polygon(drive_time_data)
-      rings = drive_time_data&.dig('geometry', 'rings')
+    def round_band(band)
+      if (band % 10).zero?
+        band
+      else
+        (band.to_i / 10) * 10
+      end
+    end
+
+    def extract_polygon(rings)
       geojson = "{\"type\":\"Polygon\",\"coordinates\":#{rings}}"
       RGeo::GeoJSON.decode(geojson)
     end
 
     def download_data
       offset = 0
       loop do
-        response = @drivetime_band_client.get_drivetime_bands(offset, 1)
+        response = @drivetime_band_client.get_drivetime_bands(offset, 30)
+
         break if response.blank?
 
         response.each(&method(:create_and_save_drive_time_data))
-        offset += 1
+        offset += 30
```

Comment:
Can we get the offset size pulled out into a constant? It is used in a couple of places and would be bad if we only updated one.